tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -32,10 +32,8 @@
 try:
     import ray
 
-    # Ray nightly version is always set to 3.0.0.dev0
-    _ray_nightly = version.parse(ray.__version__) >= version.parse("3.0.0.dev0")
-    _ray_114 = version.parse(ray.__version__) >= version.parse("1.14")
-    if _ray_114:
+    _ray_200 = version.parse(ray.__version__) >= version.parse("1.13")
```

Comment:
Why 1.13 as opposed to >= 2.0?

Reply:
@dantreiman Ah yeah good catch, that should've been > 1.13. Will update